main.go

```diff
@@ -9,18 +9,18 @@ import (
 	"github.com/golang/glog"
 	"golang.org/x/oauth2"
 
-	"github.com/google/go-github/v28/github"
+	"github.com/google/go-github/github"
```

Comment:
The v28 dit not work for me when I tested it locally. I've read in the docs of go-github that you can use either v28 or github, not 100% sure what is the difference. Happy to come back to v28 if that's a problem.

Reply:
OK. I looked at the gomod and I understand now. Reverted.